python/shotgun_desktop/upgrade_startup.py

```diff
@@ -74,14 +76,10 @@ def upgrade_startup(splash, sgtk, app_bootstrap):
 
     try:
         latest_descriptor = current_desc.find_latest_version()
-    # This happens when a proxy blocks the connection (derives from ProxyError)
-    except httplib2.socks.HTTPError:
-        logger.warning("A proxy blocked the access to tank.shotgunstudio.com.")
-        return False
     # This happens when someone runs a local server and the Desktop doesn't have
     # internet access for security reasons.
-    except httplib2.ServerNotFoundError:
-        logger.warning("Could not find tank.shotgunstudio.com. Internet connection down?")
+    except (httplib2.HttpLib2Error, httplib2.socks.HTTPError, httplib.HTTPException), e:
+        logger.warning("Could not access tank.shotgunstudio.com (%s)." % e.__class__.__name__)
```

Comment:
Should we mention what the url is for? 
Should we show any potential messages contained in the exception as well?
Also, I think standard logging prefers `,` over `%`

```
logger.warning("Could not access the Toolkit App Store at tank.shotgunstudio.com (%s): %s", 
               e.__class__.__name__, e)
```

Overall looks good though! No need to review again.